interactive/urls.py

```diff
@@ -14,6 +14,7 @@
     path("", AnalysisRequestCreate.as_view(), name="analysis-create"),
     path("<slug:slug>/", AnalysisRequestDetail.as_view(), name="analysis-detail"),
     path("<str:slug>/", AnalysisRequestDetail.as_view(), name="analysis-detail"),
+    path("<str:slug>/update", AnalysisReportUpdate.as_view(), name="report-update"),
```

Comment:
Same point about renaming to `edit` here please.  Also, the URL needs a trailing slash.

Reply:
Why is the slash needed? That seems semantically odd?

Reply:
It's consistent with how we do the rest of our page URLs :)